peering/models/models.py

```diff
@@ -22,6 +22,7 @@
     RoutingPolicyType,
 )
 from peering.fields import ASNField, CommunityField
+from peeringdb.enums import GeneralPolicy
```

Comment:
Line can be removed.

Reply:
Sigh, thought isort/black would pick this up :(